ClientCredentialsKeypairs/AuthenticationStoreDefault.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Fhi.ClientCredentialsKeypairs;
+
+/// <summary>
+/// Used in tests by different repos..
+/// </summary>
+[Obsolete("Create your own implementation of IAuthTokenStore")]
```

Comment:
Do you know why we have this AuthenticationStoreDefault @OsirisTerje ? I have made some assumptions here, but I don't think we should duplicate our code for this case.